Gemfile.lock

```diff
@@ -120,6 +120,7 @@ GEM
 
 PLATFORMS
   x86_64-darwin-20
+  x86_64-darwin-21
```

Comment:
we might want to add ruby here: 

See: https://bundler.io/man/gemfile.5.html#PLATFORMS

Reply:
I've checked `ue` and `buyer-intent-api` and both just have `ruby` in there. I suppose there are no reasons not to do the same so replacing